### DISCUSSION TITLE
Pull request checks give errors not related to pull request

### DISCUSSION THREAD

**Discussion by PapaToemmsn** [ClickHouse/ClickHouse#60397] [Q&A] [answered]
I was trying to find some information on how to handle this case but was unsuccessful. I have created a pull request and a lot of the automated checks fail but the failure has no relation to my changes (message: DB::Exception: io_uring is not supported by this system).

As the pull request is stuck now I'd like to know how to handle this case?

Thank you!

**Comment by vdimir** [comment, marked as answer]
Our CI sometimes experiences flaky or broken checks, unfortunately, due to a lot of ongoing changes. In such cases, simply tag someone who is assigned to your PR, if nobody, choose from suggested reviewers by GitHub, or if you're unsure, just tag someone you often see in PRs or in the modified code.

**Reply by PapaToemmsn** [reply]
Thank you for your answer, sorry for late reply, didn't realize that there was an answer.

**Comment by PapaToemmsn** [comment]
I continued syncing my fork with the main project thus triggering CI. At some point the errors did not happen anymore.
